### PR TITLE
Update user docu especially with regard to speed monitoring

### PR DESCRIPTION
**Pull request by HiddenBug**

### CODE REVIEW

File: prbt_hardware_support/README.md [hunk 1]
```diff
@@ -1,21 +1,75 @@
...
+  - [Definitions](#definitions)
+  - [Safe stop 1](#safe-stop-1-ss1)
+  - [Brake test](#brake-test)
+  - [Operation modes](#operation-modes)
+  - [Speed monitoring](#speed-monitoring)
+- [Overview system components](#overview-system-components)
+
+# Introduction
+The `prbt_hardware_support` package provides support for the Pilz hardware
+PNOZmulti and PSS 4000. A number of safety feature are provided which are
```
Comment (martiniil) [inline prbt_hardware_support/README.md]:
```suggestion
PNOZmulti and PSS 4000. A number of safety features are provided which are
```

File: prbt_hardware_support/README.md [hunk 2]
```diff
@@ -1,21 +1,75 @@
...
+information see section about 
+[supported DIN EN ISO 10218-1 safety features](#supported-din-en-iso-10218-1-safety-features)).  
+  
+Due to the fact, that the communication between the Pilz safety controllers 
+and ROS is based on the Modbus communication protocol, the package also 
+contains C++ files providing ROS support for the Modbus communication protocol.
+  
+In the [system overview section](#overview-system-components),
+you can find a component diagram showing the overall architecture of our
+system. The component diagram shows all our nodes and the connection
```
Comment (martiniil) [inline prbt_hardware_support/README.md]:
```suggestion
system. The component diagram shows all our nodes and the connections 
```

File: prbt_hardware_support/README.md [hunk 3]
```diff
@@ -1,21 +1,75 @@
...
+and ROS is based on the Modbus communication protocol, the package also 
+contains C++ files providing ROS support for the Modbus communication protocol.
+  
+In the [system overview section](#overview-system-components),
+you can find a component diagram showing the overall architecture of our
+system. The component diagram shows all our nodes and the connection 
+between them.
+
+**Please note:**  
+The launch files included in the `prbt_hardware_support` package, don't need to
```
Comment (martiniil) [inline prbt_hardware_support/README.md]:
```suggestion
The launch files included in the `prbt_hardware_support` package don't need to
```

File: prbt_hardware_support/README.md [hunk 4]
```diff
@@ -1,21 +1,75 @@
...
+be called directly by the user. They already are included in our overall
```
Comment (martiniil) [inline prbt_hardware_support/README.md]:
```suggestion
be called directly by the user. They already are included in our top-level
```

File: prbt_hardware_support/README.md [hunk 5]
```diff
@@ -1,21 +1,75 @@
...
+# Supported DIN EN ISO 10218-1 safety features
+The prbt_hardware_support package provides support for a number of safety
+features which are essential for a DIN EN ISO 10218-1 certifiable robot system.
+  
+**Please note:**  
+The DIN EN ISO 10218-1 support is WORK IN PROGRESS!!!  
+ 
+## Needed/supported hardware
+In order for the safety features to work, one needs certain hardware
+components supporting certain safety feature.
```
Comment (martiniil) [inline prbt_hardware_support/README.md]: suggestion:

> In order for the safety features to work, one needs special hardware
components supporting special safety features.

File: prbt_hardware_support/README.md [hunk 6]
```diff
@@ -1,21 +1,75 @@
...
+- Safety controller: PSS 4000 (with special PSS 4000 program)
+- Electronic selector switch (for operation mode): PITmode
+- Enabling switch: PITenable
+- Pushbutton unit (with emergency stop): PITgatebox
+
+**Please note:**  
+Not all safety features in this section are supported by both PILZ safety 
+controllers! Currently, the PNOZmulti does not support the brake test feature 
+(for more information about the brake test feature see the 
+[brake test section](#brake-test)).
```
Comment (martiniil) [inline prbt_hardware_support/README.md:56]: @dbakovic Do you approve the description here?

**Review by martiniil — CHANGES_REQUESTED** [review]
Just a few grammar issues.

File: prbt_hardware_support/README.md (hunk 5 above)
Comment (HiddenBug) [reply prbt_hardware_support/README.md]: Done.

File: prbt_hardware_support/README.md (hunk 6 above)
Comment (HiddenBug) [reply prbt_hardware_support/README.md:56]: @dbakovic was asked. He approved everything so far. The minor requested change was made. -> Done.

**Pull request merged**